etc/picongpu/lawrencium-lbnl/picongpu.profile.example

```diff
@@ -56,6 +56,6 @@ export MY_NAME="$(whoami) <$MY_MAIL>"
 
 # "tbg" default options #######################################################
 #   - SLURM (sbatch)
-#   - fermi queue (also available: 2 K20 via k20_profile.tpl)
+#   - fermi queue (also available: 2 K20 via k20.tpl)
 export TBG_SUBMIT="sbatch"
-export TBG_TPLFILE="etc/picongpu/lawrencium-lbnl/fermi_profile.tpl"
+export TBG_TPLFILE="etc/picongpu/lawrencium-lbnl/fermi.tpl"
```

Comment:
QUESTION:
From the diff I just see deletion of `k20_profile.tpl` and `fermi_profile.tpl` no renaming - were these accidentally deleted or were there new profiles before?

Reply:
huh, weird. good catch, will double-check and add again if necessary!

Reply:
fixed! thank you!

Reply:
:+1: